fedimint-server/src/lib.rs

```diff
@@ -334,12 +361,19 @@ impl FedimintServer {
         // process messages until new epoch or we have a proposal
         let mut outcomes: Vec<HbbftConsensusOutcome> = loop {
             match self.await_next_epoch().await? {
-                Some(msg) if self.start_next_epoch(&msg) => break self.handle_message(msg).await?,
-                Some(msg) => self.handle_message(msg).await?,
-                None => break vec![],
+                EpochTriggerEvent::NewMessage(msg) if self.start_next_epoch(&msg) => {
+                    break self.handle_message(msg).await?
+                }
+                EpochTriggerEvent::NewMessage(msg) => self.handle_message(msg).await?,
+                _ => break vec![],
             };
         };
 
+        // save any transactions we have in the channel
+        while let Ok(tx) = self.tx_receiver.try_recv() {
```

Comment:
This will swallow the `Disconnected` error, but this seems ok since the next `select!` in `await_next_epoch` will catch it.